core/src/main/scala/filodb.core/query/RangeVector.scala

```diff
@@ -296,18 +296,24 @@ final case class RawDataRangeVector(key: RangeVectorKey,
                                     partition: ReadablePartition,
                                     chunkMethod: ChunkScanMethod,
                                     columnIDs: Array[Int],
-                                    dataBytesScannedCtr: AtomicLong) extends RangeVector {
+                                    dataBytesScannedCtr: AtomicLong,
```

Comment:
It would be nice to have some comments here on what these Counters mean

Reply:
will add the proper comment in the next iteration, otherwise the approval will disappear